### PR TITLE
Add bifrost aa1 to master

### LINKED ISSUE #11: There should be a self generating GraphQL API
**Describe the feature or change you'd like**
I would like to add a self generating GraphQL API like wagtail-grapple.

It should at least support:
Query:
WagtailPage
AbstractForm
AbstractDocument
AbstractImage
AbstractRendition
BaseSetting
Snippet

Mutation:
AbstractForm

Subscription:
WagtailPage

**Additional context**
image

### PR DESCRIPTION
**Pull request by kleberbaum**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no self generating GraphQL API like wagtail-grapple. #11 

**What is the new behavior (if this is a feature change)?**
- Now there is a self generating GraphQL API like wagtail-grapple. Closes #11 

**Other information**:
- Ref: https://pypi.org/project/graphql-core/
- Ref: https://pypi.org/project/graphene-django/
- Ref: https://pypi.org/project/graphene-django-optimizer/
- Ref: https://pypi.org/project/django-graphql-jwt/
- Ref: https://pypi.org/project/channels/
- Ref: https://pypi.org/project/asgi_redis/
- Ref: https://pypi.org/project/graphql-ws/

### CODE REVIEW

File: esite/urls.py [hunk 1]
```diff
@@ -27,12 +27,15 @@
...
 
+from esite.bifrost import urls as api_urls
+
 from esite.utils.cache import get_default_cache_control_decorator
 from esite.utils.views import favicon, robots
 
 
 # Private URLs are not meant to be cached.
 private_urlpatterns = [
+    url(r"", include(api_urls)),
```
Comment (pinterid) [inline esite/urls.py:38]: Is "r" required?
```suggestion
    url("", include(api_urls)),
```

**Review by pinterid — CHANGES_REQUESTED** [review]
Just found a single thing

**Review by Aichnerc — APPROVED** [review]
Only looked at comments and general look and feel. I'm no python dev to judge any functionality or coding style. LGTM.

**Review by schettn — APPROVED** [review]
LGTM

**Review by pinterid — APPROVED** [review]
LGTM

**Review by pinterid — APPROVED** [review]
LGTM

**Review by simonprast — APPROVED** [review]
looks very good to me

**Review by schettn — APPROVED** [review]
lgtm

**Review by kleberbaum — COMMENTED** [review]
Was totally ok to merge.

**Pull request merged**